src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 * Export OpenMetrics format from Prometheus exporters ([#5107](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5107))
+* For requests with OpenMetrics format, scope info is automatically added ([#5086](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5086))
```

Comment:
@robertcoltheart Scope info is added regardless of whether OpenMetrics is added, right?

Reply:
`otel_scope_name` and `otel_scope_version` tags are added regardless of what is requested, yes. This mirrors the behavior of the Java OTEL library. The `otel_scope_info` is only added when OpenMetrics is requested, otherwise Prometheus doesn't know how to handle `info` types and will fault.

Reply:
> The otel_scope_info is only added when OpenMetrics is requested

The example that you provided in the PR description has `otel_scope_info` added for both the cases though.

Reply:
Damn, I don't know how I missed that. Sorry about that, let me raise a PR for that right now.

Reply:
No worries!

Reply:
See: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5182